README.md

```diff
@@ -18,6 +18,8 @@
   3. [Tricks](#4-tricks)  
   4. [Debugging](#5-debugging)  
   
+# Introduction
+Using bash, the built-in command-line interface in linux, offers a convenient way to navigate the file system, as well as gain access to tools native to bash, such as using pipes.
```

Comment:
Also this sentence isn't completely grammatically correct: _"Using bash, ..., offers a convenient way"_ — it's the bash itself, not using it, that offers that stuff.

Maybe rephrase like this: _"Bash, the built in ..."_,
or like this: _"Using bash, ..., is a convenient way to navigate ..."_